### ISSUE TITLE
MGs normalized metagenomics and metatranscriptomics profiles 

### ISSUE THREAD

**Issue by JSSaini** [arumugamlab/MIntO#6]
Thank you for the nice tool. Can we run only the MGs normalization step using minto? 

Reference: Read count step normalization step in the methodology of the manuscript (_MGs_ _normalization_). And the fifth step of the Results section (5. Alignment and normalization). 

I have already obtained the MAGs, and now I would like to check their 1) Abundance in metatranscriptomics dataset based on marker genes. 2) Quantification of the gene expression of respective MAGs. 

Thank you for your kind instructions. 

Regards,
Jaspreet

**Comment by arumugamlab** [comment]
Hello,

Sorry for the delayed response.

You can indeed do that. You can feed your MAGs as **reference genomes** to MIntO by naming them with `.fna` extension,  placing them in a directory and providing that path in `PATH_reference` variable. MIntO will annotate them, extract marker genes and then normalize using MGs in the `gene_abundance.smk` and `data_integration.smk` steps.

For (1), using MG normalization is what you would want. For (2), I am not sure exactly what you meant - do you mean you want to compare which MAG is generating more RNA compared to the rest? Then you would likely want total read count normalized by genome size. If you run the reference genome mode, you can get the MAG level profile for each sample at `9-mapping-profiles/refgenome-genes/<sample>/<sample>.p95.filtered.profile.relabund.prop.genome.txt.gz`. This gives you MAG relative abundance in metatranscriptomics. You could merge them together into one table and you can compare expression levels across samples. As you probably know, the issue with compositionality applies here, so comparing one species across samples needs to be done carefully. Comparing different species within a sample is more reliable.

We have just released v2.0.0-beta.1. Most functionality I mentioned above only exists in the recent version. Things should run smoothly from installation until final analysis. We have also included a test script [here](https://github.com/arumugamlab/MIntO/blob/main/testing/run_IBD_test.sh). Please clone this version using:
```
git clone https://github.com/arumugamlab/MIntO.git
git checkout tags/2.0.0-beta.1
```
and let me know if you run into issues.

Hope this helps, and please let us know if you have questions.

Best,
Mani

**Comment by arumugamlab** [comment]
Closing as there was no feedback or follow up. Please reopen if it continues to be an issue.
